src/ArgParse.jl

```diff
@@ -177,17 +178,19 @@ type ArgParseSettings
                                version::AbstractString = "Unspecified version",
                                add_help::Bool = true,
                                add_version::Bool = false,
+                               fromfile_prefix_chars = Set{Char}(),
                                autofix_names::Bool = false,
                                error_on_conflict::Bool = true,
                                suppress_warnings::Bool = false,
                                allow_ambiguous_opts::Bool = false,
                                commands_are_required::Bool = true,
                                exc_handler::Function = default_handler
                                )
-        return new(prog, description, epilog, usage, version, add_help, add_version,
-                   autofix_names, error_on_conflict, suppress_warnings, allow_ambiguous_opts,
-                   commands_are_required, copy(std_groups), "",
-                   ArgParseTable(), exc_handler)
+        return new(
+            prog, description, epilog, usage, version, add_help, add_version,
+            Set{Char}(fromfile_prefix_chars), autofix_names, error_on_conflict,
```

Comment:
We should probably make sure somewhere that the `fromfile_prefix_chars` does not contain `'-'`, as that should be reserved for options. I also wouldn't allow alphanumeric characters actually.